.golangci.yaml

```diff
@@ -75,14 +131,10 @@ linters:
     - typecheck
     - unconvert
     - unused
-    - varcheck
     - whitespace
     - bodyclose
     - noctx
     - rowserrcheck
-    - structcheck
-    - unparam
```

Comment:
Are these removals intentional?

Reply:
same question. If they are, in the future it would be good to mention in the PR description or a comment in this file

Reply:
Yes these are deprecated in new golangci-lint version. Updated the PR description